bayleef/ingest.py

```diff
@@ -355,10 +355,10 @@ def stats(arr, additional_funcs=[]):
 
     arr1 = img1_bt_projected.read_array()
     arr2 = img2_bt_projected.read_array()
-    arr1[arr1 == pysis.specialpixels.SPECIAL_PIXELS['Real']['Null']] = 0
-    arr2[arr2 == pysis.specialpixels.SPECIAL_PIXELS['Real']['Null']] = 0
-    arr1[arr1 == -32768.] = 0
-    arr2[arr2 == -32768.] = 0
+
+    arr1[arr1==0] = np.nan
+    arr1[arr1==pysis.specialpixels.SPECIAL_PIXELS['Real']['Null']] = np.nan
+    arr2[arr2==0] = np.nan
```

Comment:
Why is only arr1 getting checked for special ISIS pixels?

Reply:
Hey Kelvin, sorry for the delay in getting back to you. Just had a look through the code again and that should not be the case. Would you mind editing it so it looks like the following. Thank you!
```suggestion
    arr1[arr1==0] = np.nan
    arr1[arr1==pysis.specialpixels.SPECIAL_PIXELS['Real']['Null']] = np.nan
    arr2[arr2==0] = np.nan
    arr2[arr2==pysis.specialpixels.SPECIAL_PIXELS['Real']['Null']] = np.nan
```

Reply:
I'll be doing some Bayleef maintenance later today as part of my open source, I'll probably wrap this PR along with it.